retail-managment-system/rms-ui/src/assets/i18n/ar.json

```diff
@@ -85,13 +87,15 @@
   },
   "customrs": {
     "customer": "العملاء",
+    "employee": "موظف",
```

Comment:
add section for employee don't use costumers 

as structure we make in translation